modules/bmc/bmc_api.rb

```diff
@@ -1,30 +1,48 @@
+require 'bmc/ipmi'
```

Comment:
It's actually ok here, as bmc_api will only get loaded if the module was enabled. https://github.com/theforeman/smart-proxy/blob/develop/lib/smart_proxy.rb#L100 is the bit that takes care of this...